src/api/faultmanagement/FaultmanagementAPI.js

```diff
@@ -0,0 +1,86 @@
+/*****************************************************************************
```

Comment:
File is named with incorrect case. Should be TitleCase.

Reply:
Gotcha!